features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/attachments/preview/AttachmentsPreviewPresenter.kt

```diff
@@ -57,6 +60,20 @@ class AttachmentsPreviewPresenter @AssistedInject constructor(
 
         val ongoingSendAttachmentJob = remember { mutableStateOf<Job?>(null) }
 
+        DisposableEffect(Unit) {
```

Comment:
What if it's disposed for some reason (new screen on the backstack, config changes not handled)?

Reply:
Yes good point, so I guess I have to handle the cancellation with an event.

Reply:
62aab87c0924736cb29c3845ac94b1f8df64dcfd